README.md

```diff
@@ -107,6 +107,47 @@ xpub16y4vhpyuj2t84gh2qfe3ydng3wc37yqzxev6gce380fvvg47ye8um3dm3wn5a64gt7l0fh5j6sj
 ```
 </details>
 
+<details>
+  <summary>How to construct a script and calculate its hash</summary>
+
+  Let's say I have the following recovery phrase
+```
+$ recovery-phrase.prv
+nothing heart matrix fly sleep slogan tomato pulse what roof rail since plastic false enlist
+```
+ Now we create root key
+```
+$ cat recovery-phrase.prv | cardano-address key from-recovery-phrase Shelley > root.prv
+xprv1apjwjs3ksgm5mnnk0cc5v5emgv0hmafmmy8tffay5s2ffk69830whwznr46672ruucdzwwtv9upv72e4ylrypyz5m6cyh0p00t7n3u3agt20lv32j4kxcqlkzu78nzjx0ysxxlc2ghfz9prxfmrds802xsh67k7t
+```
+ And derive two signing keys according to multisig CIP
```

Comment:
Would be good to link to the CIP here.

Reply:
but there is not yet CIP link now, right?

Reply:
Well, that's also my point. There should be a CIP before we start implementing thins :sweat_smile: